### PR TITLE
Added footer to missing pages and updated styles

### LINKED ISSUE #573: Testimonials: Add footer to theme
Every page should have a footer!

One page that we know doesn't have a footer is the testimonials, for example:

https://ocwnext.odl.mit.edu/testimonials/
https://ocwnext.odl.mit.edu/testimonials/evelyn-laurito/

#### Acceptance Criteria:

- [ ] add a footer to both pages in the theme

#### Related issues 

- [x] #574
- [ ] standardize the quote format in testimonials 
- [ ] rename "testimonials" to "stories"

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #573 

#### What's this PR do?
Adds footer to the following pages
- course_collection page
- course_list page
- testimonials page
- testimonials list page
- resource list page

#### How should this be manually tested?
Verify the footer renders properly and is responsive
- course_collection page
- course_list page
- testimonials page
- testimonials list page
- resource list page
- home page
- course pages

#### Screenshots (if appropriate)
Screenshot 2022-03-31 at 1 11 55 PM
Screenshot 2022-03-31 at 1 02 46 PM
Screenshot 2022-03-31 at 1 02 30 PM

### CODE REVIEW

File: www/layouts/course-lists/single.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <div class="container standard-width mx-auto mt-5">
     {{ partial "course_list" . }}
   </div>
+  {{ block "footer" . }} {{ partialCached "footer" . }} {{end}}
```
Comment (MAbdurrehman12) [inline www/layouts/course-lists/single.html:10]:
```suggestion
  {{ block "footer" . }} 
      {{ partialCached "footer" . }} 
  {{end}}
```

This indentation can also be done on the other pages

**Review by MAbdurrehman12 — APPROVED** [review]
👍

File: www/layouts/newsletter/section.html [hunk 2]
```diff
@@ -447,5 +447,5 @@ <h2>Be informed</h2>
   </script>
   <!--End mc_embed_signup-->
 </div>
-
+{{ block "footer" . }} {{ partialCached "footer" . }} {{end}}
```
Comment (pdpinch) [inline www/layouts/newsletter/section.html:450]: I don't recall ever using a "newsletter" layout. Is there a screenshot for this?

**Review by pdpinch — COMMENTED** [review]
Is there no base layout that applies to all pages? It would be better to add the footer there instead of having to add it to each layout.

File: www/layouts/newsletter/section.html (hunk 2 above)
Comment (Wassaf-Shahzad) [reply www/layouts/newsletter/section.html:450]: Let me share those with you, this was one of my first PR's on this project

Screenshot 2022-03-31 at 5 18 33 PM
Screenshot 2022-03-31 at 5 18 25 PM

**Comment by Wassaf-Shahzad** [comment]
> Is there no base layout that applies to all pages? It would be better to add the footer there instead of having to add it to each layout.

@pdpinch 
The thing is we add the header to every new section.html and single.html and followed the same pattern. However there is a baseofhtml.html and adding footer there might sole the problem but we would probably need to move the header as well

**Comment by pdpinch** [comment]
Ok. I'm ok with merging this as is, but please open a tech debt issue to move the header and footer into a common base template

File: www/layouts/newsletter/section.html (hunk 2 above)
Comment (pdpinch) [reply www/layouts/newsletter/section.html:450]: of course, I remember now! 🙂

**Review by pdpinch — APPROVED** [review]
If @MAbdurrehman12 has already reviewed this, then go ahead and merge.

**Pull request merged**